app/models/roles.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
```

Comment:
Let's have a one line space after the `'use strict';` and in other files where it appears